x/data/client/tx.go

```diff
@@ -88,24 +90,31 @@ func MsgAttestCmd() *cobra.Command {
 				return err
 			}
 
-			iri := args[0]
-			if len(iri) == 0 {
-				return sdkerrors.ErrInvalidRequest.Wrap("iri is required")
-			}
-
 			attestor := clientCtx.GetFromAddress()
-			content, err := data.ParseIRI(iri)
-			if err != nil {
-				return sdkerrors.ErrInvalidRequest.Wrapf("invalid iri: %s", err.Error())
+
+			if len(args[0]) == 0 {
+				return sdkerrors.ErrInvalidRequest.Wrap("at least one iri is required")
 			}
-			graph := content.GetGraph()
-			if graph == nil {
-				return sdkerrors.ErrInvalidRequest.Wrap("can only attest to graph data types")
+
+			var hashes []*data.ContentHash_Graph
+
+			iris := strings.Split(args[0], ",")
+
+			for i := range iris {
+				content, err := data.ParseIRI(iris[i])
+				if err != nil {
+					return sdkerrors.ErrInvalidRequest.Wrapf("invalid iri: %s", err.Error())
```

Comment:
how about adding index to the error message?

Reply:
I don't think the user needs to know the index if they receive the specific iri that is invalid. Also, we've never included an index in an error message as far as I know.